salt/metalk8s/kubeadm/init/addons/files/coredns_deployment.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coredns
+  namespace: kube-system
+  labels:
+    k8s-app: kube-dns
+    kubernetes.io/name: "CoreDNS"
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      k8s-app: kube-dns
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-dns
+    spec:
+      priorityClassName: system-cluster-critical
+      tolerations:
+        - key: "CriticalAddonsOnly"
+          operator: "Exists"
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      containers:
+      - name: coredns
+        image: coredns/coredns:1.3.1
```

Comment:
This image should be part of our local registry

Reply:
yes I noticed that we have one but not the right version

Reply:
It doesn't seem to be the 'right' version either? In my `kubeadm` `dry-run` output, for K8s 1.11, it uses `coredns:1.2.2`.

Reply:
hmmm, https://github.com/kubernetes/kubernetes/issues/69983
I guess 1.2.2 is too old, found the latest tested version to be 1.3.1 on https://console.cloud.google.com/gcr/images/google_containers/GLOBAL/coredns?gcrImageListsize=50&pli=1
Tell me what do you think about the version